python/ray/serve/tests/test_kv_store.py

```diff
@@ -40,6 +41,59 @@ def test_ray_internal_kv_collisions(serve_instance):
     assert kv1.get("1") == b"1"
 
 
+def _test_operations(kv_store):
+    # Trival get & put
+    kv_store.put("1", b"1")
+    assert kv_store.get("1") == b"1"
+    kv_store.put("2", b"2")
+    assert kv_store.get("1") == b"1"
+    assert kv_store.get("2") == b"2"
+
+    # Overwrite same key
+    kv_store.put("1", b"-1")
+    assert kv_store.get("1") == b"-1"
+
+    # Get non-existing key
+    assert kv_store.get("3") is None
+
+    # Delete existing key
+    kv_store.delete("1")
+    kv_store.delete("2")
+    assert kv_store.get("1") is None
+    assert kv_store.get("2") is None
+
+    # Delete non-existing key
+    kv_store.delete("3")
```

Comment:
We should probably also run some regular integration tests with this kv store. Maybe the controller failure test?

Reply:
Sure but a simpler version given current one didn't fully pass yet ...